### PR TITLE
POL-982 Hybrid Use Benefit Policy - currency separator symbol shown as undefined

### PR DESCRIPTION
**Pull request by vadgun**
### Description

n the incident of Hybrid Use Benefit Policy, currency separator is shown as undefined:

### Issues Resolved

[POL-982](https://flexera.atlassian.net/browse/POL-982)

### Link to Example Applied Policy

[Applied policy](https://app.flexeratest.com/orgs/1105/automation/applied-policies/projects/60073?policyId=658230b65ec8620001b97922)

### Contribution Check List

- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by Rangelmv23 — APPROVED** [review]
LGTM

File: cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt [hunk 1]
```diff
@@ -476,7 +476,7 @@ script "js_ahub_recommendations", type: "javascript" do
     while (values[0].length > 3) {
       var chunk = values[0].substr(-3)
       values[0] = values[0].substr(0, values[0].length - 3)
-      formatted_number = separator + chunk + formatted_number
+      formatted_number = ds_currency['t_separator'] + chunk + formatted_number
```
Comment (night-codes) [inline cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt:479]: It is better to change this value when calling the function rather than here

**Review by night-codes — DISMISSED** [review]

**Review by XOmniverse — APPROVED** [review]
LGTM

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**